.github/workflows/publish_master.yml

```diff
@@ -58,12 +58,24 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@master
       - name: Login to ECR
-        uses: docker/login-action@v1
```

Comment:
I'm not sure why I used `docker/login-action` here instead of `aws-actions/amazon-ecr-login`. It might be because `aws-actions/amazon-ecr-login` doesn't work with `docker/setup-buildx-action`. We'll have to verify that it works without `docker/login-action`.

Reply:
> I'm not sure why I used `docker/login-action` here instead of `aws-actions/amazon-ecr-login`. It might be because `aws-actions/amazon-ecr-login` doesn't work with `docker/setup-buildx-action`. We'll have to verify that it works without `docker/login-action`.

I used `aws-actions/amazon-ecr-login` to push to the test repo and `docker/login-action` to push to the master repo and they both worked in this workflow. https://github.com/AgnostiqHQ/covalent/actions/runs/2023541511/workflow